account_invoice_triple_discount/__manifest__.py

```diff
@@ -3,13 +3,13 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Account Invoice Triple Discount",
-    "version": "16.0.3.0.0",
+    "version": "18.0.1.0.0",
     "category": "Accounting & Finance",
     "author": "QubiQ, Tecnativa, GRAP, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/account-invoicing",
     "license": "AGPL-3",
     "summary": "Manage triple discount on invoice lines",
-    "depends": ["account"],
+    "depends": ["sale", "account"],
```

Comment:
Why this change ?

Reply:
https://github.com/OCA/account-invoicing/pull/1939/files#diff-3c2a0c335b3658f70e6cc646d2f82097e866dbcd7267a0b3497df142d9cd2392R14

Odoo move group in sale